src/customize-pop/customize-desktop.md

```diff
@@ -0,0 +1,99 @@
+# Customizing the Desktop
+
+Change the function of the `SUPER` key, desktop backgrounds, workspace features, and placement and visibility of desktop components in `Settings` ➞ `Desktop`.
+
+- [`Desktop Options`](/customize-pop/customize-desktop.md#desktop-options) provides options to modify the `SUPER` key's function, enabling Hot Corner, and adding or positioning top bar elements.
```

Comment:
`enabling Hot Corner` may read better as `enabling the Hot Corner` or `enabling a Hot Corner`.

Reply:
I'm going to write these bulleted sections as imperative sentences. This omits wordy "provides options to", "allows the user to", "includes settings for" and makes these navigational sections easier to read. The "Function" column in "Option/Function" tables will also be written as imperative, unless the function describes an action performed without direct action of the user (such as the Dock auto-hiding).

Reply:
I like the rewording of the bullet points. The hot corner wording as `enable the Hot Corner` is satisfactory.